src/CommunityToolkit.Maui/Behaviors/Validators/EmailValidationBehavior.shared.cs

```diff
@@ -117,4 +115,13 @@ static string DomainMapper(Match match)
 			return match.Groups[1].Value + domainName;
 		}
 	}
+
+	[GeneratedRegex(@"^([1-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])(\.([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])){3}$", RegexOptions.None, 250)]
+	private static partial Regex ValidIpv4Regex();
+
+	[GeneratedRegex(@"^[^@\s]+@[^@\s]+\.[^@\s]+$", RegexOptions.IgnoreCase, 250)]
+	private static partial Regex ValidEmailRegex();
```

Comment:
Do we need a valid prefix?

Reply:
What do you mean? Could you share an example?

Reply:
Just EmailRegex without Valid

Reply:
Because if we have “valid” email reflex there should be invalid email reflex which doesn’t make sense. So I suggest rename it to EmailRegex.

Reply:
✅ Done [ValidEmailRegex -> EmailRegex](https://github.com/CommunityToolkit/Maui/pull/763/commits/e8042ed698ca632466d0f2b33752d9ccd14327cb)